Adapter_Revit_UI/AdapterActions/Push.cs

```diff
@@ -73,7 +72,7 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
             if (UIControlledApplication != null && pushConfig.SuppressFailureMessages)
                 UIControlledApplication.ControlledApplication.FailuresProcessing += ControlledApplication_FailuresProcessing;
 
-            
+
             // If unset, set the pushType to AdapterSettings' value (base AdapterSettings default is FullCRUD).
             if (pushType == PushType.AdapterDefault)
                 pushType = PushType.DeleteThenCreate;
```

Comment:
If you do https://github.com/BHoM/Revit_Toolkit/pull/577/files#r388440231,
then this could be put just after that, so we have an initial setup section that checks/sets the PushType.